app/models/user/rpg.rb

```diff
@@ -0,0 +1,26 @@
+module User::Rpg
+  extend ActiveSupport::Concern
+
+  def xp_needed_for_next_level
+    next_level - self.experience_points
+  end
+
+  def next_level
+    if self.level === 0
+      return 32
+    elsif self.level === 1
```

Comment:
The required xp to reach lv 2 was 0 before, so I hardcoded this part. Chose some arbitrary random numbers....

Also thought it would be cool to start at level 0.